org.flatpak.ClassicSnakeTerminal.yml

```diff
@@ -0,0 +1,50 @@
+app-id: org.flatpak.ClassicSnakeTerminal
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+modules:
+  - name: python
+    buildsystem: simple
+    build-commands:
+      - ./configure --prefix=/app
+      - make -j$(nproc)
+      - make install
+    sources:
+      - type: archive
+        url: https://www.python.org/ftp/python/2.7.18/Python-2.7.18.tar.xz
+        md5: fd6cc8ec0a78c44036f825e739f36e5a
+  - name: nodejs
+    buildsystem: simple
+    build-commands:
+      - ./configure --prefix=/app
+      - make -j$(nproc)
```

Comment:
rather than nproc could you use $FLATPAK_BUILDER_N_JOBS 

https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html there's an array of examples if you need https://github.com/search?q=org%3Aflathub+FLATPAK_BUILDER_N_JOBS&type=code